src/larceler/LarCelerStandalone.hh

```diff
@@ -0,0 +1,106 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file larceler/LarCelerStandalone.hh
+//! \note This file is an `art` plugin and should only be included by its .cc
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <memory>
+#include <vector>
+#include <art/Utilities/ToolConfigTable.h>
+
+#include "LarStandaloneRunner.hh"
+
+#include "detail/LarCelerConfig.hh"
+
+namespace sim
+{
+class SimEnergyDeposit;
+class OpDetBacktrackerRecord;
+}  // namespace sim
+
+// TODO: This will be defined upstream:
+// see https://github.com/nuRiceLab/larsim/pull/1
+namespace phot
+{
+class OpticalSimInterface
```

Comment:
I think the last bit is unifying this class with the larsim interface (e.g. `begin_job()` -> `beginJob()`).